README.md

```diff
@@ -1,2 +1,52 @@
-# styelint-config-cssrecipes
-cssrecipes shareable config for stylelint 
+# stylelint-config-cssrecipes
+
+> cssrecipes shareable config for stylelint
+
+## Install
+
+```console
+$ npm install stylelint-config-cssrecipes
+```
+
+## Usage
+
+As a JavaScript library
+
+```
+var fs = require("fs")
+var postcss = require("postcss")
+var cssnext = require("cssnext")
+var cssrecipesConfig = require("stylelint-config-cssrecipes")
+var reporter = require("postcss-reporter")
+var stylelint = require("stylelint")
+
+var input = fs.readFileSync("index.css", "utf8")
+
+var output = postcss()
+  .use(cssnext())
```

Comment:
If `cssrecipes` is just using `cssnext` then I think `stylelint` can come first in the processor pipeline, as all the syntax in the `cssnext` is supported by `stylelint`. This will allow you to turn on rules like `*-no-vendor-prefixes-*` and make use of things like `selector-root-no-composition`